new/INS1022ESZA.xml

```diff
@@ -45,7 +45,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPlace>
                 <place type="privateHouse" subtype="institution">
-                    <placeName>Private collection of Qasis ʾƎšat Zammana</placeName>
+                    <placeName>Private collection of Qasis ʾƎšat Zammana ʾAndābet</placeName>
```

Comment:
I think, we write the title "qasis" with small letters and without capitalisation.